packages/components/src/Link/Link.stories.mdx

```diff
@@ -0,0 +1,47 @@
+import { Meta, Story, Preview } from '@storybook/addon-docs/blocks';
+import { Link, Text } from '@marigold/components';
+
+<Meta title="Components/Link" />
+
+# Link
+
+## Description
+
+Element to style links. `nav` variant is for links in navigation context.
+
+## Properties
+
+| Property             | Type                    | Default  |
+| :------------------- | :---------------------- | :------- |
+| `href`               | string                  |          |
+| `variant` (optional) | `normal`, `bold`, `nav` | `normal` |
+
+## Import
+
+```tsx
+import { Link } from '@marigold/components';
+```
+
+## Usage
```

Comment:
It would be more nice if the three examples are in three different lines. But just a visual note.